src/screens/AirdropDashboard/components/AirdropInProgress.tsx

```diff
@@ -21,79 +27,50 @@ interface Props {
   subscribeWallet: SubscribeWalletActionCreator;
   error: boolean;
   loading: boolean;
+  usersQuantity: number;
 }
 
 export const AirdropInProgress: FC<Props> = props => {
-  let _carouselRef: Carousel<AirdropWalletCardData>;
+  const [communityCarouselActive, setCommunityCarouselActive] = useState(false);
+  const { usersQuantity, subscribedWallets, loading, error, availableWallets, subscribeWallet } = props;
+  let _carouselRef: Carousel<AirdropCarouselCardData>;
 
-  const setRef = (carouselRef: Carousel<AirdropWalletCardData>) => {
+  const setRef = (carouselRef: Carousel<AirdropCarouselCardData>) => {
     _carouselRef = carouselRef;
   };
 
-  const AirdropInProgressContent: FC<Props> = ({
-    error,
-    loading,
-    subscribedWallets,
-    availableWallets,
-    subscribeWallet,
-  }) => {
-    if (loading) {
-      return (
-        <View style={styles.loadingContainer}>
-          <Loading />
-        </View>
-      );
-    }
-    if (error) {
-      return (
-        <View style={styles.errorContainer}>
-          <Error />
-        </View>
-      );
-    }
-
-    return (
-      <>
-        {subscribedWallets?.length > 0 ? (
-          <>
-            <AirdropWalletsCarousel styles={styles.carouselStyles} items={subscribedWallets} setRef={setRef} />
-            <View style={styles.walletsListContainer}>
-              <AirdropWalletsList
-                wallets={subscribedWallets}
-                title={i18n.airdrop.dashboard.registeredWallets}
-                itemCallToAction={(wallet: Wallet) => (
-                  <RegisteredWalletAction onActionClick={() => setCarouselActiveElement(wallet)} />
-                )}
-              />
-            </View>
-          </>
-        ) : (
-          <>
-            <Image source={images.airdrop} style={styles.airdropImage} />
-            <Text style={styles.description}>{i18n.airdrop.dashboard.desc2}</Text>
-          </>
-        )}
-        {availableWallets?.length > 0 && (
-          <View style={styles.walletsListContainer}>
-            <AirdropWalletsList
-              wallets={availableWallets}
-              title={i18n.airdrop.dashboard.availableWallets}
-              itemCallToAction={(wallet: Wallet) => (
-                <AvailableWalletAction onActionClick={() => subscribeWallet(wallet)} />
-              )}
-            />
-          </View>
-        )}
-      </>
-    );
-  };
-
   const setCarouselActiveElement = (wallet: Wallet) => {
-    const snapIndex = props.subscribedWallets.findIndex(w => w.id === wallet.id && w.balance === wallet.balance);
+    const snapIndex = subscribedWallets.findIndex(w => w.id === wallet.id && w.balance === wallet.balance);
```

Comment:
why do we compare balance? id won't be enough?

Reply:
done